horizontalintegration/xmc-hztl-foundation-2024-q1
# Feature FIS-365 Sitecore JSS v22.1.3 Upgrade

## What does this PR do and why?

This PR upgrades the Sitecore JSS dependency from v21.6.0 to v22.1.3.

## Screenshots or screen recordings

N/A

## How to set up and validate locally

1. Check out the branch _feature/FIS-365-sitecore-jss-22_1_.
2. From a command prompt, navigate to _./src/hztl-foundation_.
3. From the same command prompt, execute `npm rm -r ./node_modules` to delete the existing _node_modules_ folder.
4. Again from the same command prompt, execute `npm install` to [re]install all project dependencies.
5. Again from the same command prompt, execute `npm run start:connected`, then, from a browser, navigate to https://localhost:3000 and confirm that the project, pages, and components function as expected.

## PR acceptance checklist

This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
